e2e/test_lint_requirements.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# -*- indent-tabs-mode: nil; tab-width: 2; sh-indentation: 2; -*-
+
+# This script acts as an e2e test for lint_requirements command of fromager
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "$SCRIPTDIR/common.sh"
+
+pass=true
+
+# Test to demonstrate that command works as expected when input files are valid
+
+if ! fromager lint-requirements "$SCRIPTDIR/validate_inputs/constraints.txt" "$SCRIPTDIR/validate_inputs/requirements.txt"; then
+    echo "the input files are not correctly formatted." 1>&2
```

Comment:
It would be helpful to phrase these messages in a way that it's clear that this is the unexpected result. In this case, something like "the input files should have been recognized as correctly formatted".

Reply:
Okay, fixed in latest push.